### DISCUSSION TITLE
Links to meeting pages reference old domain name

### DISCUSSION THREAD

**Discussion by honorcoding** [code4recovery/12-step-meeting-list#516] [Q&A] [answered]
We recently migrated our staging site (staging.al-anon-ak.org) to live (al-anon-ak.org). When we go to the meetings page, https://al-anon-ak.org/meetings/, all the links on that page reference the old url. (e.g. staging.al-anon-ak.org, not al-anon-ak.org). I ran a search replace on the database and this did not fix it. I cannot see anywhere in the website itself that references staging.al-anon-ak.org. Please assist.

**Comment by lovethecodelabs** [comment]
go through the wp-posts table changing "staging." to "" in the following 
fields: content, guid, and clear cache if necessary.

This should work

**Comment by honorcoding** [comment]
I tried that. Didn't work.

On 8/29/2021 3:00 PM, Matthew wrote:
> go through the wp-posts table changing "staging." to "" in the following
> fields: content, guid, and clear cache if necessary.
>
> This should work
>
>
>
>
> On 2021-08-29 14:58, honorcoding wrote:
> > We recently migrated our staging site (staging.al-anon-ak.org) to live
> > (al-anon-ak.org). When we go to the meetings page,
> > https://al-anon-ak.org/meetings/, all the links on that page reference
> > the old url. (e.g. staging.al-anon-ak.org, not al-anon-ak.org). I ran
> > a search replace on the database and this did not fix it. I cannot see
> > anywhere in the website itself that references staging.al-anon-ak.org.
> > Please assist.
> >
> > --
> > You are receiving this because you are subscribed to this thread.
> > Reply to this email directly, view it on GitHub [1], or unsubscribe
> > [2].
> > Triage notifications on the go with GitHub Mobile for iOS [3] or
> > Android [4].
> >
> > Links:
> > ------
> > [1]
> > https://github.com/code4recovery/12-step-meeting-list/discussions/516
> > [2]
> > 
> https://github.com/notifications/unsubscribe-auth/AAFHIHYQDEIPOH6E7IDKLNDT7KUOVANCNFSM5DAVDRCQ
> > [3]
> > 
> https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&amp;mt=8&amp;pt=524675
> > [4]
> > 
> https://play.google.com/store/apps/details?id=com.github.android&amp;referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub 
> <https://github.com/code4recovery/12-step-meeting-list/discussions/516#discussioncomment-1251381>, 
> or unsubscribe 
> <https://github.com/notifications/unsubscribe-auth/AVMYBCVUTEQ7M7DNJ7N4TBLT7K32DANCNFSM5DAVDRCQ>.
> Triage notifications on the go with GitHub Mobile for iOS 
> <https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675> 
> or Android 
> <https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub>. 
>
>
-- 
John Schultz
Honor Coding
907-350-4520
***@***.***

**Comment by lovethecodelabs** [comment]
ALso, Change also Site URL & WordPress URL in Settings and save.

**Comment by lovethecodelabs** [comment]
It is possible there is a hardcoded URL in wp-settings.php, to check as 
well.

**Comment by lovethecodelabs** [comment]
https://wordpress.org/support/article/changing-the-site-url/

is a good summary, too.

**Comment by honorcoding** [comment]
Go to the website: https://al-anon-ak.org. All the links work just fine.

Now go to the meetings page: https://al-anon-ak.org/meetings/

If I try to click on a link it pulls up the old url with "staging." in it.

For example:

- Brown Bag (Monday) takes me to 
"https://staging.al-anon-ak.org/meetings/brown-bag-2/". This page does 
not exist.
- But if you remove the "staging." from the url and just try that - it 
pulls up the correct page. See: https://al-anon-ak.org/meetings/brown-bag-2/

The problem is that the meetings list  (found on 
https://al-anon-ak.org/meetings/) is somehow using a 'staging.' url. 
Where is that URL coming from? It does not appear to be the permalink 
for the Meetings custom post type.

There isn't a problem with the Meeting CPT - it appears to be with that 
meetings listing page.


On 8/30/2021 10:43 AM, Matthew wrote:
> https://wordpress.org/support/article/changing-the-site-url/
>
> is a good summary, too.
>
>
>
> On 2021-08-30 11:39, ***@***.*** wrote:
> > It is possible there is a hardcoded URL in wp-settings.php, to check as
> > well.
> >
> >
> >
> >
> > On 2021-08-30 11:35, ***@***.*** wrote:
> >> ALso, Change also Site URL & WordPress URL in Settings and save.
> >>
> >>
> >>
> >> On 2021-08-30 11:21, honorcoding wrote:
> >>> I tried that. Didn't work.
> >>>
> >>> On 8/29/2021 3:00 PM, Matthew wrote:
> >>>> go through the wp-posts table changing "staging." to "" in the
> >>> following
> >>>> fields: content, guid, and clear cache if necessary.
> >>>>
> >>>> This should work
> >>>>
> >>>>
> >>>>
> >>>>
> >>>> On 2021-08-29 14:58, honorcoding wrote:
> >>>> > We recently migrated our staging site (staging.al-anon-ak.org) to
> >>> live
> >>>> > (al-anon-ak.org). When we go to the meetings page,
> >>>> > https://al-anon-ak.org/meetings/, all the links on that page
> >>> reference
> >>>> > the old url. (e.g. staging.al-anon-ak.org, not al-anon-ak.org). I
> >>> ran
> >>>> > a search replace on the database and this did not fix it. I cannot
> >>> see
> >>>> > anywhere in the website itself that references
> >>> staging.al-anon-ak.org.
> >>>> > Please assist.
> >>>> >
> >>>> > --
> >>>> > You are receiving this because you are subscribed to this thread.
> >>>> > Reply to this email directly, view it on GitHub [1], or
> >>> unsubscribe
> >>>> > [2].
> >>>> > Triage notifications on the go with GitHub Mobile for iOS [3] or
> >>>> > Android [4].
> >>>> >
> >>>> > Links:
> >>>> > ------
> >>>> > [1]
> >>>> >
> >>> https://github.com/code4recovery/12-step-meeting-list/discussions/516
> >>>> > [2]
> >>>> >
> >>>>
> >>> 
> https://github.com/notifications/unsubscribe-auth/AAFHIHYQDEIPOH6E7IDKLNDT7KUOVANCNFSM5DAVDRCQ
> >>>> > [3]
> >>>> >
> >>>>
> >>> 
> https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&amp;mt=8&amp;pt=524675
> >>>> > [4]
> >>>> >
> >>>>
> >>> 
> https://play.google.com/store/apps/details?id=com.github.android&amp;referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub
> >>>>
> >>>> —
> >>>> You are receiving this because you authored the thread.
> >>>> Reply to this email directly, view it on GitHub
> >>>>
> >>> 
> <https://github.com/code4recovery/12-step-meeting-list/discussions/516#discussioncomment-1251381>,
> >>>
> >>>> or unsubscribe
> >>>>
> >>> 
> <https://github.com/notifications/unsubscribe-auth/AVMYBCVUTEQ7M7DNJ7N4TBLT7K32DANCNFSM5DAVDRCQ>.
> >>>> Triage notifications on the go with GitHub Mobile for iOS
> >>>>
> >>> 
> <https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675>
> >>>
> >>>> or Android
> >>>>
> >>> 
> <https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub>.
> >>>
> >>>>
> >>>>
> >>> --
> >>> John Schultz
> >>> Honor Coding
> >>> 907-350-4520
> >>> ***@***.***
> >>>
> >>> --
> >>> You are receiving this because you commented.
> >>> Reply to this email directly, view it on GitHub [1], or unsubscribe
> >>> [2].
> >>> Triage notifications on the go with GitHub Mobile for iOS [3] or
> >>> Android [4].
> >>>
> >>> Links:
> >>> ------
> >>> [1]
> >>> 
> https://github.com/code4recovery/12-step-meeting-list/discussions/516#discussioncomment-1255557
> >>> [2]
> >>> 
> https://github.com/notifications/unsubscribe-auth/AAFHIH5PXLLRVN2AV2PS2E3T7PD3RANCNFSM5DAVDRCQ
> >>> [3]
> >>> 
> https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&amp;mt=8&amp;pt=524675
> >>> [4]
> >>> 
> https://play.google.com/store/apps/details?id=com.github.android&amp;referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub 
> <https://github.com/code4recovery/12-step-meeting-list/discussions/516#discussioncomment-1255639>, 
> or unsubscribe 
> <https://github.com/notifications/unsubscribe-auth/AVMYBCS5WXISQV5OKVU2XL3T7PGL5ANCNFSM5DAVDRCQ>.
> Triage notifications on the go with GitHub Mobile for iOS 
> <https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675> 
> or Android 
> <https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub>. 
>
>
-- 
John Schultz
Honor Coding
907-350-4520
***@***.***

**Comment by lovethecodelabs** [comment]
You may have to re-save on Permalinks page if all other settings are 
correct.

**Comment by honorcoding** [comment]
I tried that as well.

It appears that all the links on the site are working except the links 
on the meeting page.


On 8/30/2021 11:56 AM, Matthew wrote:
> You may have to re-save on Permalinks page if all other settings are
> correct.
>
>
>
> On 2021-08-30 11:57, honorcoding wrote:
> > Go to the website: https://al-anon-ak.org. All the links work just
> > fine.
> >
> > Now go to the meetings page: https://al-anon-ak.org/meetings/
> >
> > If I try to click on a link it pulls up the old url with "staging." in
> > it.
> >
> > For example:
> >
> > - Brown Bag (Monday) takes me to
> > "https://staging.al-anon-ak.org/meetings/brown-bag-2/". This page does
> >
> > not exist.
> > - But if you remove the "staging." from the url and just try that - it
> >
> > pulls up the correct page. See:
> > https://al-anon-ak.org/meetings/brown-bag-2/
> >
> > The problem is that the meetings list (found on
> > https://al-anon-ak.org/meetings/) is somehow using a 'staging.' url.
> > Where is that URL coming from? It does not appear to be the permalink
> > for the Meetings custom post type.
> >
> > There isn't a problem with the Meeting CPT - it appears to be with
> > that
> > meetings listing page.
> >
> > On 8/30/2021 10:43 AM, Matthew wrote:
> >> https://wordpress.org/support/article/changing-the-site-url/
> >>
> >> is a good summary, too.
> >>
> >>
> >>
> >> On 2021-08-30 11:39, ***@***.*** wrote:
> >> > It is possible there is a hardcoded URL in wp-settings.php, to
> > check as
> >> > well.
> >> >
> >> >
> >> >
> >> >
> >> > On 2021-08-30 11:35, ***@***.*** wrote:
> >> >> ALso, Change also Site URL & WordPress URL in Settings and save.
> >> >>
> >> >>
> >> >>
> >> >> On 2021-08-30 11:21, honorcoding wrote:
> >> >>> I tried that. Didn't work.
> >> >>>
> >> >>> On 8/29/2021 3:00 PM, Matthew wrote:
> >> >>>> go through the wp-posts table changing "staging." to "" in the
> >> >>> following
> >> >>>> fields: content, guid, and clear cache if necessary.
> >> >>>>
> >> >>>> This should work
> >> >>>>
> >> >>>>
> >> >>>>
> >> >>>>
> >> >>>> On 2021-08-29 14:58, honorcoding wrote:
> >> >>>> > We recently migrated our staging site
> > (staging.al-anon-ak.org) to
> >> >>> live
> >> >>>> > (al-anon-ak.org). When we go to the meetings page,
> >> >>>> > https://al-anon-ak.org/meetings/, all the links on that page
> >> >>> reference
> >> >>>> > the old url. (e.g. staging.al-anon-ak.org, not
> > al-anon-ak.org). I
> >> >>> ran
> >> >>>> > a search replace on the database and this did not fix it. I
> > cannot
> >> >>> see
> >> >>>> > anywhere in the website itself that references
> >> >>> staging.al-anon-ak.org.
> >> >>>> > Please assist.
> >> >>>> >
> >> >>>> > --
> >> >>>> > You are receiving this because you are subscribed to this
> > thread.
> >> >>>> > Reply to this email directly, view it on GitHub [1], or
> >> >>> unsubscribe
> >> >>>> > [2].
> >> >>>> > Triage notifications on the go with GitHub Mobile for iOS [3]
> > or
> >> >>>> > Android [4].
> >> >>>> >
> >> >>>> > Links:
> >> >>>> > ------
> >> >>>> > [1]
> >> >>>> >
> >> >>>
> > https://github.com/code4recovery/12-step-meeting-list/discussions/516
> >> >>>> > [2]
> >> >>>> >
> >> >>>>
> >> >>>
> >>
> > 
> https://github.com/notifications/unsubscribe-auth/AAFHIHYQDEIPOH6E7IDKLNDT7KUOVANCNFSM5DAVDRCQ
> >> >>>> > [3]
> >> >>>> >
> >> >>>>
> >> >>>
> >>
> > 
> https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&amp;mt=8&amp;pt=524675
> >> >>>> > [4]
> >> >>>> >
> >> >>>>
> >> >>>
> >>
> > 
> https://play.google.com/store/apps/details?id=com.github.android&amp;referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub
> >> >>>>
> >> >>>> —
> >> >>>> You are receiving this because you authored the thread.
> >> >>>> Reply to this email directly, view it on GitHub
> >> >>>>
> >> >>>
> >>
> > 
> <https://github.com/code4recovery/12-step-meeting-list/discussions/516#discussioncomment-1251381>,
> >> >>>
> >> >>>> or unsubscribe
> >> >>>>
> >> >>>
> >>
> > 
> <https://github.com/notifications/unsubscribe-auth/AVMYBCVUTEQ7M7DNJ7N4TBLT7K32DANCNFSM5DAVDRCQ>.
> >> >>>> Triage notifications on the go with GitHub Mobile for iOS
> >> >>>>
> >> >>>
> >>
> > 
> <https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675>
> >> >>>
> >> >>>> or Android
> >> >>>>
> >> >>>
> >>
> > 
> <https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub>.
> >> >>>
> >> >>>>
> >> >>>>
> >> >>> --
> >> >>> John Schultz
> >> >>> Honor Coding
> >> >>> 907-350-4520
> >> >>> ***@***.***
> >> >>>
> >> >>> --
> >> >>> You are receiving this because you commented.
> >> >>> Reply to this email directly, view it on GitHub [1], or
> > unsubscribe
> >> >>> [2].
> >> >>> Triage notifications on the go with GitHub Mobile for iOS [3] or
> >> >>> Android [4].
> >> >>>
> >> >>> Links:
> >> >>> ------
> >> >>> [1]
> >> >>>
> >>
> > 
> https://github.com/code4recovery/12-step-meeting-list/discussions/516#discussioncomment-1255557
> >> >>> [2]
> >> >>>
> >>
> > 
> https://github.com/notifications/unsubscribe-auth/AAFHIH5PXLLRVN2AV2PS2E3T7PD3RANCNFSM5DAVDRCQ
> >> >>> [3]
> >> >>>
> >>
> > 
> https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&amp;mt=8&amp;pt=524675
> >> >>> [4]
> >> >>>
> >>
> > 
> https://play.google.com/store/apps/details?id=com.github.android&amp;referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub
> >>
> >> —
> >> You are receiving this because you authored the thread.
> >> Reply to this email directly, view it on GitHub
> >>
> > 
> <https://github.com/code4recovery/12-step-meeting-list/discussions/516#discussioncomment-1255639>,
> >
> >> or unsubscribe
> >>
> > 
> <https://github.com/notifications/unsubscribe-auth/AVMYBCS5WXISQV5OKVU2XL3T7PGL5ANCNFSM5DAVDRCQ>.
> >> Triage notifications on the go with GitHub Mobile for iOS
> >>
> > 
> <https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675>
> >
> >> or Android
> >>
> > 
> <https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub>.
> >
> >>
> >>
> > --
> > John Schultz
> > Honor Coding
> > 907-350-4520
> > ***@***.***
> >
> > --
> > You are receiving this because you commented.
> > Reply to this email directly, view it on GitHub [1], or unsubscribe
> > [2].
> > Triage notifications on the go with GitHub Mobile for iOS [3] or
> > Android [4].
> >
> > Links:
> > ------
> > [1]
> > 
> https://github.com/code4recovery/12-step-meeting-list/discussions/516#discussioncomment-1255692
> > [2]
> > 
> https://github.com/notifications/unsubscribe-auth/AAFHIH6G4QMNKMQ3ASPI57TT7PIBDANCNFSM5DAVDRCQ
> > [3]
> > 
> https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&amp;mt=8&amp;pt=524675
> > [4]
> > 
> https://play.google.com/store/apps/details?id=com.github.android&amp;referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub 
> <https://github.com/code4recovery/12-step-meeting-list/discussions/516#discussioncomment-1255890>, 
> or unsubscribe 
> <https://github.com/notifications/unsubscribe-auth/AVMYBCS7GH2IKDQJDGNAYOTT7PO5FANCNFSM5DAVDRCQ>.
> Triage notifications on the go with GitHub Mobile for iOS 
> <https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675> 
> or Android 
> <https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub>. 
>
>
-- 
John Schultz
Honor Coding
907-350-4520
***@***.***

**Comment by lovethecodelabs** [comment]
Looks like it's fixed

Sent from my T-Mobile 5G Device
________________________________

**Reply by tech2serve** [reply]
Hi @honorcoding —

Can you right up what you did to fix this? Or if one of @lovethecodelabs’ answers worked, mark that as the correct answer? This will help others who may have the similar issue.

Thanks!

**Comment by honorcoding** [comment, marked as answer]
I don't know what fixed it. I did everything that was recommended, but 
none of that seemed to work. Then all of the sudden it was working. I 
think it might have been some lingering cache somewhere. I really have 
no idea.


On 9/1/2021 1:59 PM, Tech2Serve wrote:
>
> Hi @honorcoding <https://github.com/honorcoding> —
>
> Can you right up what you did to fix this? Or if one of 
> @lovethecodelabs <https://github.com/lovethecodelabs>’ answers worked, 
> mark that as the correct answer? This will help others who may have 
> the similar issue.
>
> Thanks!
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub 
> <https://github.com/code4recovery/12-step-meeting-list/discussions/516#discussioncomment-1268875>, 
> or unsubscribe 
> <https://github.com/notifications/unsubscribe-auth/AVMYBCVQRQDAXXSLKHI3XKLT72O3LANCNFSM5DAVDRCQ>.
> Triage notifications on the go with GitHub Mobile for iOS 
> <https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675> 
> or Android 
> <https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub>. 
>
>
-- 
John Schultz
Honor Coding
907-350-4520
***@***.***
